packages/react-composites/src/composites/CallWithChatComposite/CallWithChatComposite.tsx

```diff
@@ -341,7 +337,7 @@ const CallWithChatScreen = (props: CallWithChatScreenProps): JSX.Element => {
   /* @conditional-compile-remove(PSTN-calls) */
   const alternateCallerId = callAdapter.getState().alternateCallerId;
   /* @conditional-compile-remove(close-captions) */
-  const isTeamsCall = useSelector(getIsTeamsCall);
+  const isTeamsCall = callAdapter.getState().isTeamsCall;
```

Comment:
Will this value always keep unchanged or not? If this could be potentially changed during the call, we should still use selectors

Reply:
The value will keep unchanged since it's indicating if its currently a teams call or ACS call